### ISSUE TITLE
Microsoft resets the default app for microsoft-edge files after restart

### ISSUE THREAD

**Issue by TheBooker66** [da2x/EdgeDeflector#136]
I'll start by saying I don't expect anything to be done with this, just notifying the devs.

**To Reproduce**
Restart your computer.

**Expected behavior**
Nothing.

**Actual behavior/describe the bug:**
![image](https://user-images.githubusercontent.com/83127216/134717679-49b03d25-bb4b-45e8-b9b1-d792b3ba590b.png)

**Details:**
 - Windows version: 21H1 (Build 19043.1237)

This is disgraceful behaviour by Microsoft.

**Comment by da2x** [comment]
Hi, thanks for the report. Please help me gather more information about the issue.


* Had you just completed a Windows Update that prompted you to use Microsoft recommended browser settings?
* Is EdgeDeflector still installed?
* Can you change the settting back to EdgeDeflector?
* Have you got the Brave browser installed?
* What version of EdgeDeflector is instslled? (Check the file properties of the .exe.)
* Anything in Windows Security: Protection history?
* Is there anything in the Windows Event Viewer? I dont know exacrly what you're looking for, but you know the approximate time it happen.
* The Diagnostic Data Viewer will likely contain the most useful information, but it isn't enabled by default. If you happen to have it enabled, then pleaee check it as well.

**Comment by TheBooker66** [comment]
Hey. as per your request, here are my answers:

- I haven't completed a Windows update, just restarted my computer after 8 days.
- It was still installed when I reported the bug (I later uninstalled it as I realised I don't actually need it)
- I could change it back.
- I don't use Brave. My main (and pretty much only) browser is Opera GX.
- Unknown, but I assume 1.2.3 as I installed it recently.
- I didn't see anything unusual or relating to EdgeDeflector in windows security.
- Sorry, I couldn't find anything related.
- I don't have it enabled, but I enabled it now. Thx!
